packages/@aws-cdk/aws-apigateway/test/authorizers/integ.cognito-authorizer.ts

```diff
@@ -0,0 +1,37 @@
+import * as cognito from '@aws-cdk/aws-cognito';
+import { App, Stack } from '@aws-cdk/core';
+import { AuthorizationType, CognitoUserPoolsAuthorizer, MockIntegration, PassthroughBehavior, RestApi } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * get the IdToken for the created pool by adding user/app-client and using aws cognito-idp
```

Comment:
Would be nice to add the exact CLI command to get the id token here.

Otherwise, the user making changes here and running this test has to go fishing through docs.